source/reference/minio-mc-admin/mc-admin-user-svcacct-add.rst

```diff
@@ -0,0 +1,122 @@
+.. _minio-mc-admin-svcacct-add:
+
+=============================
+``mc admin user svcacct add``
+=============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin user svcacct add
+
+
+Syntax
+------
+
+.. start-mc-admin-svcacct-add-desc
+
+The :mc-cmd:`mc admin user svcacct add` command adds a new access key to an existing MinIO or AD/LDAP user.
+
+.. end-mc-admin-svcacct-add-desc
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command creates a new access key associated to an existing MinIO user:
+
+      .. code-block:: shell                                                                 
+         :class: copyable
+
+         mc admin user svcacct add                       \
+            --access-key "myuserserviceaccount"          \                                  
+            --secret-key "myuserserviceaccountpassword"  \
+            --policy "/path/to/policy.json"              \
+            myminio myuser                                                                  
+                                                                                   
+      The command returns the access key and secret key for the new account.
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] admin user svcacct add     \
+                                     [--access-key]  \
+                                     [--secret-key]  \
+                                     [--policy]      \
+                                     [--comment]     \
+                                     ALIAS           \
+                                     USER
+					
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+.. mc-cmd:: USER
+   :required:
+
+   The name of the user to which MinIO adds the new access key.
+
+   - For :ref:`MinIO-managed users <minio-users>`, specify the access key for the user.
+   - For :ref:`Active Directory/LDAP users <minio-external-identity-management-ad-ldap>`, specify the Distinguished Name of the user.
+   - For :ref:`OpenID Connect users <minio-external-identity-management-openid>`, use the :ref:`MinIO Console <minio-console>` to generate access keys.
```

Comment:
I'd suggest adding a note to this effect after line 23.

Reply:
@djwfyi I think you mean referring people to the console if they want access keys for OpenID Connect users? (And not this command at all.)

Reply:
I mean specifically line 77.
It's worth mentioning at the top of the page, I think.

Reply:
I'm making it an admonition. Since it's pretty much "You need to go to this other place, not here."